containers/podman-desktop-extension-ai-lab
# Adopt ErrorMessage from ui-svelte library

### What does this PR do?

### What issues does this PR fix or reference?

Fixes #971 

### How to test this PR?

Check error when 
- creating an inference server
- choosing a model to import
- creating a playground
